### PR TITLE
l18n to Portuguese (Brazil)

### PR DESCRIPTION
**Pull request by jppcel**

### CODE REVIEW

File: i18n/pt-br.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  name: 'Português (Brasil)',
+  key: 'pt-br',
```
Comment (yuryleb) [inline i18n/pt-br.js]: Maybe it's better to use 'pt-BR' key to exactly match osrm-text-instructions' language pack name?

**Review by yuryleb — CHANGES_REQUESTED** [review]
Also it's necessary to update `src/localization.js` to explicitly load your localization file.

**Comment by jppcel** [comment]
done.

**Review by yuryleb — APPROVED** [review]
Thanks!

**Comment by bbridges** [comment]
I'm not able to start the `osrm/osrm-frontend` container from this PR since the filename used in `src/localization.js` was assuming the path was case-insensitive.

I get a build error with:
```text
> browserify -d src/index.js -s osrm > bundle.raw.js && uglifyjs bundle.raw.js -c -m --source-map=bundle.js.map -o bundle.js

Error: Cannot find module '../i18n/pt-BR' from '/src/src'
```

It runs fine with `'../i18n/pt-br'`, though.

**Comment by freenerd** [comment]
@bbridges should be fixed via #298

**Pull request merged**